src/pages/settings/InitialSettingsPage.tsx

```diff
@@ -367,6 +369,11 @@ function InitialSettingsPage({session, userWallet, bankAccountList, fundList, wa
     const currentUserDetails = currentUserPersonalDetails;
     const avatarURL = currentUserDetails?.avatar ?? '';
     const accountID = currentUserDetails?.accountID ?? '-1';
+    const usernameContainEmojis = CONST.REGEX.EMOJIS.test(currentUserPersonalDetails?.displayName ?? '');
+    let processedTextArray: TextWithEmoji[] = [];
+    if (usernameContainEmojis) {
```

Comment:
Please go through your PR and make sure all boolean variable names start start with `is`, `are`, `has`, `was`, `does`, etc...